src/Symfony/Component/Form/FormEvents.php

```diff
@@ -12,7 +12,7 @@
 
 /**
  * To learn more about how form events work check the documentation
- * entry at {@link http://symfony.com/doc/any/components/form/form_events.html}
+ * entry at {@link http://symfony.com/doc/any/components/form/form_events.html}.
```

Comment:
This "short description" should be part of the long description. That's why my fixer did that.